### PR TITLE
ACS Event Reference Documentation

### PR DESCRIPTION
**Pull request by phuo1032**
Forgot this when we released ACS.

### CODE REVIEW

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# Adobe Campaign Standard event reference
+
+## Events handled
+
+### Campaign Request Content  <a id="configuration-response-content"></a>
+
+This event is dispatched from Event Hub for the following scenarios:
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: - Please add "the" between "from" and "Event Hub".

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+1. Event data containing loaded consequences for the latest registered campaign rules needs to be dispatched
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please change to "Dispatch the event data that contains the loaded consequences for the latest registered Campaign rules."

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+```
+
+
+### Configuration Response Content  <a id="configuration-response-content"></a>
+
+The data property in this event is used by each extension to modify its current settings. Each extension is responsible for reading out the part of the data property for which it is concerned.
+
+#### Data payload definition  <a id="data-payload-definition-3"></a>
+
+The Adobe Campaign Standard extension will read the following key from the configuration event:
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please change "will read" to "reads"

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+| `__stage__campaign.server` | String | Yes | This contains the endpoint URL for the stage environment Adobe Campaign Standard instance |
+| `__stage__campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in stage environment Adobe Campaign Standard |
+| `campaign.server` | String | Yes | This contains the endpoint URL for the production environment Adobe Campaign Standard instance |
+| `campaign.pkey` | String | Yes | This contains the identifier for a mobile app configured in production environment Adobe Campaign Standard |
+| `campaign.mcias` | String | Yes | This contains the in-app messaging service URL endpoint |
+| `campaign.timeout` | String | Yes | This contains the amount of time to wait for a response from in-app messaging service |
+| `global.privacy` | Boolean | Yes | This contains the mobile privacy status settings |
+
+{% hint style="info" %}
+If `global.privacy` is changed to **optout**, linkage fields will be reset. All downloaded messages, rules will be erased and no tracking request may leave the device.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please reword as follows:

"After `global.privacy` is changed to **optout**, the linkage fields are reset. All downloaded messages and rules are erased, and no tracking request can leave the device."

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+    "campaign.pkey":"",
+    "__dev__campaign.server":"mcias.campaign-demo.adobe.com",
+    "__stage__campaign.server":"mcias.campaign-demo.adobe.com",
+    "campaign.server":"mcias.campaign-demo.adobe.com"
+}
+```
+
+### Request Identity  <a id="request-identity"></a>
+
+This event is dispatched by public API to setLinkageFields to set the linkage fields. It is used to download personalized In-App messages.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please reword as follows:

"This event is dispatched by the API to setLinkageFields, which sets the linkage fields. It is used to download personalized in-app messages."

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+    "starttimestampmillis" :44,    
+    "maxsessionlength":22
+}
+```
+
+
+## Events dispatched
+
+### Campaign Response Content
+This event is a response from the Adobe Campaign Standard extension contains the message interaction details (Viewed, Triggered or Clicked).
```
Comment (rymorale) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: I think we can add an 'and' to make the sentence a bit clearer: This event is a response from the Adobe Campaign Standard extension and contains the message interaction details (Viewed, Triggered or Clicked).

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+        "firstName":"john", 
+        "lastName":"doe", 
+        "age":30
+    }
+}
+```
+
+### Request Reset  <a id="request-reset"></a>
+
+This event is dispatched when a user decides they no longer wants to utilize personalized In-App messages. Following the recieval of this event, campaign deletes all personalized messages and redownloads only generic messages.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please reword as follows:

"This event is dispatched when users no longer want to use personalized in-app messages. After this event is received, Campaign deletes all personalized messages and downloads only generic messages again."

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md (hunk 6 above)
Comment (rekrishn) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-event-reference.md]: Please add "and" between "extension" and "contains"

**Pull request merged**